### DISCUSSION TITLE
Feature Request: Ability to filter star page by 'list status'

### DISCUSSION THREAD

**Discussion by spmadden** [community/community#64218] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

First up: I love the ability to organize my repository stars into lists - good job so far.

It would be very helpful to be able filter the main star page to hide all stars that already exist in one of my lists.  This way I can use the main star page as a "misc/unsorted" list.

![image](https://github.com/community/community/assets/498181/543e026d-67dd-4794-823d-a498f2309368)

**Comment by Khoivu369** [comment]
All

**Comment by 7807917212** [comment]
********

**Comment by Smart007optics** [comment]
All
